python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/test_asset_daemon_perf.py

```diff
@@ -350,7 +350,7 @@ def do_scenario(self: "PerfScenario"):
     PerfScenario(
         snapshot=unpartitioned_500_assets_2_random_runs,
         n_freshness_policies=0,
-        max_execution_time_seconds=5,
+        max_execution_time_seconds=10,
```

Comment:
ominous addition that was needed (test case takes about 6 seconds now) - I imagine because we are not caching intermediate results from the recursive calls anymore

Reply:
wouldn't the visited set serve the same purpose as the cached method?

Reply:
not exactly - before we were also caching each recursive call, now we are not